docs/community/release.md

```diff
@@ -1,6 +1,32 @@
 ---
-title: Release Manager
+title: Release Manage
 sidebar_position: 5
 ---
 
-tbd...
+KubeEdge releases currently happen approximately four times per year. 
```

Comment:
every three months should be more accurate?

Reply:
> every three months should be more accurate?

done.